btree/btree.c

```diff
@@ -1778,24 +1872,25 @@ static int64_t node_get(struct node_op *op, struct td *tree,
 		m0_rwlock_init(&node->n_lock);
 		op->no_node = node;
 		nt->nt_opaque_set(addr, node);
+		m0_rwlock_write_unlock(&lru_lock);
+
+		tree_lock(tree, lock_acquired);
+		ndlist_tlink_init_at(op->no_node, &tree->t_active_nds);
+		tree_unlock(tree, lock_acquired);
 	}
 
 	return nxt_state;
 }
 
 #ifndef __KERNEL__
 /**
- * This function decrements the reference count for this node and if the
- * reference count reaches '0' then the node is made available for future
- * mode_get requests.
+ * This function decrements the reference count for this node descriptor and if
+ * the reference count reaches '0' then the node descriptor is moved to LRU
+ * list.
  *
- * @param op load operation to perform.
- * @param tree pointer to tree whose node is to be loaded or NULL if tree has
- *             not been loaded.
- * @param addr node address in the segment.
- * @param nxt state to return on successful completion
+ * @param node node descriptor.
+ * @param tree pointer to tree.
  *
- * @return next state
  */
 static void node_put(struct node_op *op, struct nd *node, bool lock_acquired,
 		     struct m0_be_tx *tx)
```

Comment:
Comment given for node_put parameters and the actual parameters dont match.
Is this expected?

Reply:
Done.